avocado/core/settings.py

```diff
@@ -172,10 +173,15 @@ def __init__(self, config_path=None):
             config_local = os.path.exists(config_path_local)
             config_intree = os.path.exists(config_path_intree)
             config_intree_extra = os.path.exists(_config_path_intree_extra)
-            if (not config_system) and (not config_local) and (not config_intree):
+            config_pkg_base = os.path.join('etc', config_filename)
+            config_pkg_exists = resource_exists('avocado', config_pkg_base)
+            config_pkg = resource_filename('avocado', config_pkg_base)
```

Comment:
Would you please modify this check to look more like what's already there?

```python
config_pkg_base = os.path.join('etc', config_filename)
if resource_exists('avocado', config_pkg_base):
    config_pkg = resource_filename('avocado', config_pkg_base)
else:
    config_pkg = None
```

Reply:
This won't work because `config_pkg` with a valid content (non-None) is used in the exception bellow.

Reply:
I see, on second look the current style is `config_$name` and `config_path_$name`, while you add `config_$name_exists` and `config_$name`. Would you please use the existing style (or change the rest of the variables?) It's easier to follow when all variables follow the same style.